### PR TITLE
Test for header related functino in http.Response

### PR DESCRIPTION
**Pull request by chunseoklee**
1. setHeader
2. removeHeader
3. getHeader

IoT.js-DCO-1.0-Signed-off-by: Chunseok Lee chunseok.lee@samsung.com

### CODE REVIEW

File: test/run_pass/test_http_header.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  };
+  res.on('end', endHandler);
+
+  res.on('data', function(chunk){
+    res_body += chunk.toString();
+  });
+};
+
+var req = http.request(options, postResponseHandler);
+req.end();
```
Comment (ILyoan) [inline test/run_pass/test_http_header.js]: Need  check on 'exit' event to see if `postResponseHandler` was actually processed.

**Comment by ILyoan** [comment]
Related to #238

**Comment by chunseoklee** [comment]
@ILyoan Code for checking req finish on exit added.

**Comment by ILyoan** [comment]
@chunseoklee LGTM

**Comment by chunseoklee** [comment]
Merged

**Pull request closed without merging**